numpy/lib/tests/test_format.py

```diff
@@ -837,10 +837,12 @@ def test_bad_header():
     assert_raises(ValueError, format.read_array_header_1_0, s)
 
     # headers without the exact keys required should fail
-    d = {"shape": (1, 2),
-         "descr": "x"}
-    s = BytesIO()
-    format.write_array_header_1_0(s, d)
+    # d = {"shape": (1, 2),
+    #      "descr": "x"}
+    s = BytesIO(
+        b"\x93NUMPY\x01\x006\x00{'descr': 'x', 'shape': (1, 2), }" +
+        b"                    \n"
+    )
```

Comment:
Did you want to add another test here, since this doesn't seem to be used at all?

Reply:
This replaces a test that used to be there before. However, before that array above was created by `write_array`, which is not possible anymore, as it is too invalid to not cause `write_array` to raise an exception (keyword `fortran_order` is missing). I had some thoughts about it in the striked-through comment above.

Reply:
Sorry, I apparently was lacking coffee and just misread the test...